gen_planet.py

```diff
@@ -1,31 +1,126 @@
 import random
-import requests
 import sys
+from abc import abstractmethod, ABCMeta
+from majormode.utils.namegen import NameGeneratorFactory
 
 
-def gen_name():
-    url = 'https://donjon.bin.sh/name/rpc-name.fcgi?type=Human+Male&n=10'
-    resp = requests.get(url)
-    names = resp.text.split('\n')
-    choice = random.choice(names)
-    return f'{"Name:":<20}{choice}'
+class PlanetAttribute(metaclass=ABCMeta):
```

Comment:
personally i'm not a huge fan of ABCs almost ever. I find they typically overcomplicate things. I know they _do_ have reasons to exist but by way of style I personally don't really like them /shrug